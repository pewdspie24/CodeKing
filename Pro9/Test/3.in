10
732880 805280
563705 887496
894392 987554
822301 914235
634562 832251
454929 953909
258347 774589
754143 837484
579193 788028
825009 896512
