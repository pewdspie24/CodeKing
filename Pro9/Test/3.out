832
6660
1349
1899
3667
10341
13628
1574
4249
1436
